### ISSUE TITLE
Version of Orthofinder?

### ISSUE THREAD

**Issue by skitchen19** [mpdunne/orthofiller#10]
I have tried to run OrthoFiller.py (v1.1.4) with OrthoFinder version 2.2.7 and 2.3.3 with the sample data. In both cases after OthoFinder finishes I get the following:
`output_folder/sequences/aa
Can't find Orthogroup output file... exiting...`

I can confirm that the output directory "aa" contains the OrthoFinder output directory. 
`2445872 Dec  9 13:33 Ashgo1_1_AssemblyScaffolds.fasta.aa.fasta
3150717 Dec  9 13:33 Debha1_AssemblyScaffolds.fasta.aa.fasta
2579674 Dec  9 13:34 Klula1_AssemblyScaffolds.fasta.aa.fasta
      4096 Dec  9 13:34 OrthoFinder
3023784 Dec  9 13:33 Sacce_S288C.genome.nomt.fa.aa.fasta
3292120 Dec  9 13:33 Yarli1_AssemblyScaffolds.fasta.aa.fasta
`
I believe this is a OrthoFinder version difference because the newer versions of do not produce OrthologousGroups.**csv** or Orthogroups.csv files but OrthologousGroups.**tsv** and  Orthogroups.**tsv**.

Can you provide the best version  of OrthoFinder to use with OrthoFiller?

**Comment by xonq** [comment]
You are correct about this error. I have an updated version of OrthoFiller at https://gitlab.com/xonq/OrthoFiller that addresses these dependency issues and others.
